src/utils/changelog.ts

```diff
@@ -300,7 +300,7 @@ export async function generateChangesetFromGit(
         `${milestone.title}${milestone.state === 'OPEN' ? ' (ongoing)' : ''}`
       )
     );
-    if (milestone.description !== '') {
+    if (milestone.description && milestone.description !== '') {
```

Comment:
Both null and empty string are false-y, so I think this is enough:

```suggestion
    if (milestone.description) {
```

Reply:
I wanted it to be explicit but you are right.